inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/utils/CommonUtils.java

```diff
@@ -60,4 +63,26 @@ public static boolean contentsEqual(Set<?> setA, Set<?> setB) {
         }).findFirst().orElse(null);
     }
 
+    /**
+     * returnts the signature for the given method for the given method or constructor, does not contain the return type.
+     *
+     * @param m the method to query the signature for
+     * @return the signature string
+     */
+    public static String getSignature(MethodDescription m) {
```

Comment:
is this only for debugging purposes or?

Reply:
can we confirm in a test that what is produced here is corresponding the byte buddy `Advice.Origin("#m#s")`, actually there as to be some piece of code in the butebuddy that can produce what we want here as they are doing it for the Advice already?